modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4Utils.java

```diff
@@ -55,6 +57,19 @@ public static void setup() {
 
     }
 
+    private static AtomicBoolean isAvailableProcessorsSet = new AtomicBoolean();
+
+    /**
+     * Set the number of available processors that Netty uses for sizing various resources (e.g., thread pools).
```

Comment:
> I presume you do that because the plugin is loaded multiple times during testing.

This method is invoked twice in the course of normal startup, once when we bootstrap transport and again when we bootstrap HTTP.

> Is it worth logging something if we skip it?

I don't think so it's expected that the method is invoked twice?